releases/data-center/syskit-point-2024-1/point-2024-1-41-release-note.md

```diff
@@ -0,0 +1,95 @@
+---
+description: >-
+  This article lists improvements and bug fixes in Syskit Point version 2024.1.41
+---
+
+# Syskit Point 2024.1.41
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2024.1.41
+
+**Build number:** 74
+
+**Release date:** February 1, 2024
+
+## Get the Latest Version
+
+* **New to Syskit Point?** Get the latest Syskit Point version from [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit\_point).\
+  Find all steps described in detail in the [deployment article](../../../set-up-point-data-center/deployment/deploy-syskit-point.md).
+* **Already using Syskit Point?** Upgrade to the latest Syskit Point version.\
+  [![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FARMTemplates%2FPointUpdateDeploy%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade Syskit Point in the [upgrade article](../../../set-up-point-data-center/deployment/upgrade-syskit-point.md).
+
+## Features
+
+* **New updates added to the Storage Management feature**!
+    * When the storage cleanup actions take a while to complete, the progress can now be tracked. 
+    * Collaborators can now access Storage Metrics reports for their workspaces. 
+
+
+## Improvements & Bug Fixes
+
```

Comment:
Dodao bih pod bugfixes i ovo što je ušlo zadnjih dana (al nema u generiranim release notesima)
Fixed an issue where, in specific Azure regions, Syskit Point deployment or upgrade would fail with the following error:
`The parameter 'MinTlsCipherSuite' has an invalid value.`